stable/external-dns/templates/deployment.yaml

```diff
@@ -97,12 +97,16 @@ spec:
             value: "{{ .Values.cloudflare.email }}"
         {{- end }}
         {{- $root := . -}}
-        {{- range $key, $value := .Values.extraEnv }}
-          - name: {{ $key }}
+        {{- range .Values.extraEnv }}
```

Comment:
Why not make it simpler and more flexible like so:

```yaml
{{ tpl .Values.extraEnv . | indent 10 }}
```

Reply:
Because the chart stores environment variables in a secret and I didn't want to break that, although I don't know why it does that in the first place. See `secret.yaml`. I'd also be happy to remove that, if necessary.